test/parallel/test-fs-whatwg-url.js

```diff
@@ -72,3 +72,149 @@ if (common.isWindows) {
     message: `File URL host must be "localhost" or empty on ${os.platform()}`
   }));
 }
+
+// path ending with forward slash are not permitted
+const urlWithSlash = common.isWindows ?
+  new URL('file:///c:/tmp/test/') : new URL('file:///tmp/test/');
+
+const urlWithoutSlash = common.isWindows ?
+  new URL('file:///c:/tmp/test') : new URL('file:///tmp/test');
+
+fs.readFile(urlWithSlash, common.expectsError({
+  code: 'ERR_INVALID_FILE_URL_PATH',
+  type: Error,
+  message: 'File URL path illegal operation on a directory'
```

Comment:
This message seems to contain some syntax errors. Maybe `'File URL path cannot be a directory'` ?